Bottom: 4e9194a5a0a2bc77521aca4860a58388450bcb87
Top:    99d6ceb52424564e16ce746eb863530b508f6a28
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-09 17:07:06 -0500

Add parameter for buildbot version



---
diff --git a/docker/buildbot-master/Dockerfile b/docker/buildbot-master/Dockerfile
index 521c823..dde9cdb 100644
--- a/docker/buildbot-master/Dockerfile
+++ b/docker/buildbot-master/Dockerfile
@@ -1,4 +1,5 @@
-FROM        buildbot/buildbot-master:v0.9.13
+ARG         BUILDBOT_VERSION=latest
+FROM        buildbot/buildbot-master:$BUILDBOT_VERSION
 MAINTAINER  Andrew Cooper <andrew.cooper@novatechweb.com>
 
 WORKDIR /buildbot
